i2c_manager/Kconfig

```diff
@@ -0,0 +1,116 @@
+menu "I2C Port Settings"
+    depends on LV_I2C && !HAVE_I2C_MANAGER
+
+	menu "I2C Port 0"
+
+		config I2C_MANAGER_0_ENABLED
+			bool "Enable I2C port 0"
+
+		if I2C_MANAGER_0_ENABLED
+			config I2C_MANAGER_0_SDA
+				int "SDA (GPIO pin)"
+				range 0 39 if IDF_TARGET_ESP32
+				range 0 46 if IDF_TARGET_ESP32S2
+				range 0 21 if IDF_TARGET_ESP32C3
```

Comment:
Shall I make it so for all I2C pins?

Reply:
Or even for all GPIO in the whole thing?

Reply:
IMO, for all gpios. Let's assume that users know what they are doing. And in case they don't, esp-idf drivers have safety checks for this, so there will be no silent failures.

Reply:
I'll do it for I2C Manager here and then we'll do a separate PR later for the rest. We've got to finish this one at some point.